src/ScheduleFunctions.cpp

```diff
@@ -2172,12 +2053,15 @@ class RemoveLoopsOverOutermost : public IRMutator2 {
     }
 };
 
-Stmt schedule_functions(const vector<Function> &outputs,
-                        const vector<vector<string>> &fused_groups,
-                        const map<string, Function> &env,
-                        const Target &target,
-                        bool &any_memoized) {
+bool group_should_be_inlined(const vector<Function> &funcs) {
+    return funcs.size() == 1
```

Comment:
Following style more similar to rest of compiler:
```
return (a &&
        b &&
        (c || d) &&
        e);

Reply:
outer parenthesis necessary?

Reply:
I like it because it makes the indentation after the line break more editor-setting-independent.

Reply:
Ok, done